analyzers/nodejs/testdata/normal/npm-ls-json.json

```diff
@@ -0,0 +1,23 @@
+{
+  "dependencies": {
+    "a": {
+      "version": "1.0",
```

Comment:
[nit] standard semver has three fields `<major>.<minor>.<patch>`. `npm`/`nodejs` doesn't enforce this, so it should be okay